data-plane/config/source/500-dispatcher.yaml

```diff
@@ -31,6 +31,7 @@ spec:
       name: kafka-source-dispatcher
       labels:
         app: kafka-source-dispatcher
+        app.kubernetes.io/component: kafka-dispatcher
```

Comment:
@pierDipi did you mean to add this label to this Deployment file or the [new SS file](https://github.com/knative-sandbox/eventing-kafka-broker/blob/main/data-plane/config/sourcev2/500-dispatcher.yaml) ?

Reply:
Absolutely, good catch!

/hold